LivingCity/bTraffic/bPMTrafficPerson.cpp

```diff
@@ -213,6 +213,12 @@ void BPMTrafficPerson::randomPerson(int p, BTrafficPeople &people,
     people.T[p] = 2.0f + 3.0f * (((float)qrand()) / RAND_MAX); //time heading 2.0-3s
   }
 
+
+  if (carType == 2) { //bike car

```

Comment:
@luze, what's this for? Did you add this when we need a bike? Should we delete this for now? AFAIK our simulation is not using this cpp file